dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1021,6 +1031,9 @@
       "allOf": [
         {
           "$ref": "#/definitions/AnalyzeTextTaskResult"
+        },
+        {
+          "$ref": "#/definitions/DocumentDetectedLanguage"
```

Comment:
is the placement at this level a mistake? I believe it should be at the document level instead.

Reply:
Yes, you're right. Thanks for catching this. I saw you already created another PR to fix, thank you! :)